src/vs/workbench/contrib/chat/browser/chatInputPart.ts

```diff
@@ -730,16 +744,43 @@ export class ChatInputPart extends Disposable implements IHistoryNavigationWidge
 				this.attachButtonAndDisposables(widget, index, attachment, hoverDelegate);
 			}
 
+			await Promise.all(attachmentInitPromises);
+			if (store.isDisposed) {
+				return;
+			}
+
+			if (file) {
+				widget.style.cursor = 'pointer';
+				store.add(dom.addDisposableListener(widget, dom.EventType.CLICK, (e: MouseEvent) => {
```

Comment:
Title says double-click but this is just a single click? Need keyboard handling too and this should maybe become a button or something

Reply:
I was just copying what was already in the rendered version, created https://github.com/microsoft/vscode/issues/230951 to follow up